### PR TITLE
Add step to publish to PyPI if build succeeds

### PR DESCRIPTION
**Pull request by magdapoppins**
For pushes to master where there is a tag, build and push to PyPI.

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  Publish:
+    runs-on: ubuntu-20.04
+    #if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')# Todo add after testing!
+    steps:
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      - uses: actions/checkout@v2
+      - run: pip install -U pip setuptools build
```
Comment (akx) [inline .github/workflows/publish.yml:20]:
```suggestion
      - run: pip install -U pip setuptools build twine
```

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - run: python -m build .
+      - run: pip install twine
+      - run: |
+          TWINE_USERNAME="__token__" \
+          TWINE_PASSWORD=${{ secrets.TESTPYPI_API_TOKEN }} \
+            exec twine upload --verbose --repository testpypi dist/*
```
Comment (akx) [inline .github/workflows/publish.yml:26]:
```suggestion
      - run: twine upload --verbose --repository testpypi dist/*
        env:
          TWINE_USERNAME: __token__"
          TWINE_PASSWORD: ${{ secrets.TESTPYPI_API_TOKEN }}
```

File: .github/workflows/publish.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: Publish
+'on':
+  push:
+    branches:
+      - master
```
Comment (akx) [inline .github/workflows/publish.yml:5]: Shouldn't this be removed after testing too?

**Review by akx — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yml (hunk 3 above)
Comment (magdapoppins) [reply .github/workflows/publish.yml:5]: Wonder if there is a on tag option...

Comment (akx) [reply .github/workflows/publish.yml:5]: https://github.com/omerbenamram/pyevtx-rs/pull/13/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR5-R7

**Pull request merged**